### ISSUE TITLE
Website that lists all APIs available?

### ISSUE THREAD

**Issue by philstraforelli** [STAT545-UBC/Discussion#446]
As asked in class, what is the website that lists all APIs that are available online?

**Comment by alsemm** [comment]
I found this: http://www.programmableweb.com/category/all/apis

**Comment by jennybc** [comment]
@aammd Can you remind us of the website you use to learn if the data you seek is available via API? Is it the one @alsemm mentions above?

**Comment by aammd** [comment]
Yep that's it! they have a Twitter account too https://twitter.com/programmableweb
